src/common/transformations/src/transformations/common_optimizations/lin_op_sequence_fusion.cpp

```diff
@@ -87,6 +87,12 @@ ov::pass::AddAddFusion::AddAddFusion() {
         auto add1 = label_to_output[m_add1].get_node_shared_ptr();
         auto add2 = label_to_output[m_add2].get_node_shared_ptr();
 
+        for (int i = 0; i < 2; ++i) {
+            if (add1->get_input_element_type(i) == element::f64 || add2->get_input_element_type(i) == element::f64) {
```

Comment:
Can this be done in a way that it checks not a precision, but can the transformation be applied? For example using `has_evaluate` or something?